lektor/admin/static/js/views/EditPage.jsx

```diff
@@ -100,11 +100,11 @@ class EditPage extends RecordComponent {
     });
   }
 
-  setFieldValue(field, value) {
+  setFieldValue(field, value, uiChange) {
```

Comment:
We could make the default a bit more explicit by having this as `uiChange = false`. This would also make the comment below unnecessary.

Reply:
Changed. Still thinking in pre ES2015 days :)